src/_posts/2025-01-30-new-year-new-pycon.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: The PyConUK team
+title: New Year, New PyCon UK!
+---
+
+{% assign links = site.data.links.md.short %}
+
+We’re very excited to be kicking off 2025 with an exciting announcement… PyCon
+UK is back, and we’re moving to a brand-new location!
+
+After several fantastic years in Cardiff, the 2025 conference is heading up
+north. We’re delighted to announce that PyCon UK 2025 will be held in
+Manchester from the 19th to the 22nd of September. Known for its rich
+industrial history, vibrant culture, and thriving tech scene, Manchester
+provides the perfect backdrop for the conference.
+
+{% if site.venu_confirmed == true %}
+The venue for this year will be [Contact], a leading national theatre and arts
+centre. Contact works with local communities, as well as nationally and
+internationally, to provide life-changing opportunities for the next generation
+of creative leaders, artists and audiences. Located in the heart of Manchester,
+Contact offers state-of-the-art facilities and a welcoming, accessible space
+designed to facilitate collaboration, learning, and innovation.
+{% else %}
+**We expect to be able to announce our venue soon on this blog.** Keep checking
```

Comment:
Further to this, I'm not sure that the blog post itself should change content after it's posted- that'd be unusual behaviour for a blog post I think?

Reply:
Agreed. Will be changed.